eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgChannelsDb.scala

```diff
@@ -109,6 +110,8 @@ class PgChannelsDb(implicit ds: DataSource, lock: PgLock) extends ChannelsDb wit
           statement.executeUpdate("CREATE INDEX local_channels_type_idx ON local.channels ((json->>'type'))")
           statement.executeUpdate("CREATE INDEX local_channels_remote_node_id_idx ON local.channels ((json->'commitments'->'remoteParams'->>'nodeId'))")
           statement.executeUpdate("CREATE INDEX htlc_infos_idx ON local.htlc_infos(channel_id, commitment_number)")
+          statement.executeUpdate("CREATE INDEX created_timestamp_idx ON local.channels(created_timestamp)")
+          statement.executeUpdate("CREATE INDEX closed_timestamp_idx ON local.channels(closed_timestamp)")
```

Comment:
Please remove those indices, this feature doesn't justify a performance hit on the channels table (same for `SqliteChannelsDb`).